### PR TITLE
Use a tag to refer to setup-swift action version

### LINKED ISSUE #12678: Swift article points to outdated version of setup-swift action
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-swift

### What part(s) of the article would you like to see updated?

The code examples under the ["Specifying a Swift version"](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-swift#specifying-a-swift-version) headline points to an outdated version (a specific git commit) of the [fwal/setup-swift](https://github.com/marketplace/actions/setup-swift) Action.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by fwal**
### Why:

Closes #12678

### What's being changed:

I changed the version of the [fwal/setup-swift](https://github.com/marketplace/actions/setup-swift) Action to point to version tag `v1` instead.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@fwal 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Claudio060320 — COMMENTED** [review]
Duplicate of #

**Comment by ethomson** [comment]
Hi @fwal - thanks for the PR.  We pin to SHAs for third-party actions, both in the [starter workflow templates](http://github.com/actions/starter-workflows) and the docs.  Can we PIN the to the current `v1` SHA in the docs instead of `@v1`?  Thanks!

File: content/actions/automating-builds-and-tests/building-and-testing-swift.md [hunk 1]
```diff
@@ -86,10 +86,10 @@ jobs:
         swift: ["5.2", "5.3"]
     runs-on: {% raw %}${{ matrix.os }}{% endraw %}
     steps:
-      - uses: fwal/setup-swift@d43a564349d1341cd990cfbd70d94d63b8899475
+      - uses: fwal/setup-swift@v1
```
Comment (ethomson) [inline content/actions/automating-builds-and-tests/building-and-testing-swift.md:89]:
```suggestion
      - uses: fwal/setup-swift@234ae35a6c75c32600301c1072285a33503c7a69
```

File: content/actions/automating-builds-and-tests/building-and-testing-swift.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ You can configure your job to use a single specific version of Swift, such as `5
 {% raw %}
 ```yaml{:copy}
 steps:
-  - uses: fwal/setup-swift@d43a564349d1341cd990cfbd70d94d63b8899475
+  - uses: fwal/setup-swift@v1
```
Comment (ethomson) [inline content/actions/automating-builds-and-tests/building-and-testing-swift.md:106]:
```suggestion
  - uses: fwal/setup-swift@234ae35a6c75c32600301c1072285a33503c7a69
```

File: content/actions/automating-builds-and-tests/building-and-testing-swift.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ You can use the same commands that you use locally to build and test your code u
 ```yaml{:copy}
 steps:
   - uses: actions/checkout@v2
-  - uses: fwal/setup-swift@d43a564349d1341cd990cfbd70d94d63b8899475
+  - uses: fwal/setup-swift@v1
```
Comment (ethomson) [inline content/actions/automating-builds-and-tests/building-and-testing-swift.md:122]:
```suggestion
  - uses: fwal/setup-swift@234ae35a6c75c32600301c1072285a33503c7a69
```

**Review by ethomson — CHANGES_REQUESTED** [review]

File: content/actions/automating-builds-and-tests/building-and-testing-swift.md (hunk 1 above)
Comment (Claudio060320) [reply content/actions/automating-builds-and-tests/building-and-testing-swift.md:89]:
```suggestion
      - uses: fwal/setup-swift@v1
```

**Comment by fwal** [comment]
@ethomson Hi! Sure, pinned to latest version 1.11.0 🙂

**Review by lucascosti — APPROVED** [review]
Sorry for the delay, @fwal! This looks good (shas are the same as 1.11.0), so I'll merge this in. 🚀

**Pull request merged**